### PR TITLE
Auton

### PR DESCRIPTION
**Pull request by Noah9847**

### CODE REVIEW

File: 2024 Code/src/main/deploy/pathplanner/navgrid.json
Comment (Anirudh4Neutrino) [inline 2024 Code/src/main/deploy/pathplanner/navgrid.json:1]: :(

Comment (Noah9847) [reply 2024 Code/src/main/deploy/pathplanner/navgrid.json:1]: :)

Comment (ViennaRossmanith) [reply 2024 Code/src/main/deploy/pathplanner/navgrid.json:1]: :/

File: 2024 Code/src/main/java/frc/robot/subsystems/SwerveSubsystem.java [hunk 1]
```diff
@@ -51,7 +70,43 @@ public class SwerveSubsystem extends SubsystemBase {
 
   double cycle = 0;
 
+  Field2d field = new Field2d();
```
Comment (Anirudh4Neutrino) [inline 2024 Code/src/main/java/frc/robot/subsystems/SwerveSubsystem.java:73]: initialize in constructor

**Review by Anirudh4Neutrino — APPROVED** [review]
good

**Pull request merged**